app/src/main/java/org/simple/clinic/di/PagingModule.kt

```diff
@@ -0,0 +1,33 @@
+package org.simple.clinic.di
+
+import androidx.paging.Config
+import androidx.paging.PagedList
+import dagger.Module
+import dagger.Provides
+import javax.inject.Named
+
+@Module
+class PagingModule {
+
+  @Provides
+  @Named("for_measurement_history")
```

Comment:
We should perhaps start using `Qualifiers` later on to be more type-safe, WDYT?

Reply:
Good idea, I haven't worked with them before. Maybe you could create an issue on GitHub which describes what this would look like and we can decide on an approach?

Reply:
Will do that 👍🏾